### DISCUSSION TITLE
File dragndrop produces  paste: Error executing lua: on  Win11

### DISCUSSION THREAD

**Discussion by gerroon** [LazyVim/LazyVim#1919] [Q&A]
Hi

I installed it using the docs, when I drag and drop on Nvim running in Powershell, I get this error. What can I do to get it working?

`paste: Error executing lua: vim/_editor.lua:282: Vim:E21: Cannot make changes, 'modifiable' is off^@stack traceback:^@^I[C]: in function 'nvim_put'^@^Ivim/_editor.lua:282: in function <vim/_editor.lua:229>`

**Comment by dpetka2001** [comment]
I believe the drag-n-drop functionality is for GUI Neovim versions. You might be better off trying Neovide or Neovim-qt or something like that.

**Reply by gerroon** [reply]
Ok I am using Neovim-qt, and drag and drop does not work at all. The file is not opened when dropped on it. Does lazyvim need to be configured for the gui version as well? I just started using it so not know all the details well eyt.

**Reply by dpetka2001** [reply]
I don't think so, because the drag-n-drop functionality depends on how it's implemented by the frontend client I believe. You would be better off asking a question in the repo of the client you use. You can also not install Lazyvim and try to see if drag-n-drop works to make sure if it's a Lazyvim problem or not.

**Reply by gerroon** [reply]
Thanks for your help
